contracts/interfaces/ITradingTypes.sol

```diff
@@ -25,5 +24,8 @@ interface ITradingTypes {
 	struct Settlement {
 		AssetAndAmount[] toTrader;
 		AssetAndAmount[] toBroker;
+		// proofHash is to avoid tampering
+		// with the orders and order responses in the proof
+		bytes32 proofHash;
```

Comment:
I think it is better to avoid state channels terminology ("proof"). What do you think about calling it `orderHistoryHash` or `orderChecksum`.